code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1436,8 +1436,17 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			if(prob(I.force * 2)) //blood spatter!
 				bloody = TRUE
 				var/turf/location = H.loc
-				if(istype(location))
-					H.add_splatter_floor(location)
+				if(prob(50))	//hippie start -- add blood splattering to walls and stuff
```

Comment:
why did you keep the hippie start comments

Reply:
can you thanos snap them out of here, please?

Reply:
i missed that hold on